docs/content_management/field_types/field_type_reference/taxonomyentryfield.md

```diff
@@ -0,0 +1,80 @@
+# TaxonomyEntry Field Type
+
+TaxonomyEntry is a general purpose Field Type that can store only one taxonomy entry (tag or product type). 
+It is used as a parent while creating a tag or category.
+
+| Name           | Internal name         | Expected input |
+|----------------|-----------------------|----------------|
+| `TaxonomyEntry`| `ibexa_taxonomy_entry`| `TaxonomyEntry`|
+
+## PHP API Field Type 
+
+### Input expectations
+
+Field type `TaxonomyEntry` accepts Ibexa\Contracts\Taxonomy\Value\TaxonomyEntry object.
+
+| Type            | Example         |
+|-----------------|-----------------|
+| `TaxonomyEntry` | See below. |
+
+Example array:
+
+``` php
+new FieldType\TaxonomyEntry\Value(
+	new TaxonomyEntry(
+            1,
+            'foobar',
+            'Foobar',
+            'eng-GB',
+            [
+                'eng-GB' => 'Foobar',
+            ],
+            null,
+            new Content(),
+            'tags',
+        )
+);
```

Comment:
```suggestion
new FieldType\TaxonomyEntry\Value(
	new TaxonomyEntry(
            ...
        )
);
```
There is no need to manually construct TaxonomyEntry object because this is normally fetched from `TaxonomyService`.